shared/fs/footer/download.js

```diff
@@ -34,7 +38,7 @@ const Download = (props: DownloadProps) => (
       </Box>
     </ClickableBox>
     <ClickableBox style={stylesIconBox} onClick={props.isDone ? props.dismiss : props.cancel}>
-      <Icon type="iconfont-remove" style={stylesIconRight} />
+      <Icon type="iconfont-remove" {...propsIconRight} />
```

Comment:
i think you can just remove `propsIconRight` and inline `color` and `fontSize`, like with the `Icon` instance above